packages/client/hmi-client/src/components/widgets/tera-import-github-file.vue

```diff
@@ -99,13 +101,13 @@
 									:key="index"
 									@click="previewTextFile(content)"
 								>
-									<input
-										type="checkbox"
-										:id="content.name"
-										:value="content"
+									<Checkbox
```

Comment:
Out of curiosity, what is the difference here between a `Checkbox` and a `input type="checkbox`?

Reply:
one is a `primevue` component and the latter is the native version.